src/libstd/net/tcp.rs

```diff
@@ -323,6 +350,19 @@ impl TcpListener {
     /// This function will block the calling thread until a new TCP connection
     /// is established. When established, the corresponding `TcpStream` and the
     /// remote peer's address will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::net::TcpListener;
+    ///
+    /// let listener = TcpListener::bind("127.0.0.1:8080").unwrap();
+    /// if let Ok((_socket, addr)) = listener.accept() {
+    ///     println!("new client: {:?}", addr);
+    /// } else {
+    ///     println!("cannot get new client :'(");
```

Comment:
What is a scenario where calling `accept()` would result in `Err`? Maybe this error message in the `println` could describe that scenario